nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -157,9 +158,16 @@ def _permuted_ols_on_chunk(scores_original_data, tested_vars, target_vars,
     target_vars : array-like, shape=(n_samples, n_targets)
       fMRI data. F-ordered for efficient computations.
 
+    thread_id : int
+        process id, used for display.
```

Comment:
But if we have multiple threads, the display will become a bit  messy ?

Reply:
With multiple threads, the output looks like (on the console in a Jupyter notebook):

```
...
Job #1, processed 2500/5000 permutations (50.00%, 50 seconds remaining)
Job #2, processed 2530/5000 permutations (50.60%, 49 seconds remaining)
Job #1, processed 2510/5000 permutations (50.20%, 50 seconds remaining)
Job #2, processed 2540/5000 permutations (50.80%, 49 seconds remaining)
Job #1, processed 2520/5000 permutations (50.40%, 50 seconds remaining)
Job #1, processed 2530/5000 permutations (50.60%, 49 seconds remaining)
Job #2, processed 2550/5000 permutations (51.00%, 49 seconds remaining)
Job #1, processed 2540/5000 permutations (50.80%, 49 seconds remaining)
...
```

which is not the best I agree, but I'm not sure we can do much better with simple prints. 

One way to get something cleaner is to use `tqdm`, but I don't think it's worth adding a dependency for this... Here is an example:

```python
from time import sleep
from tqdm import tqdm, trange
from joblib import Parallel, delayed

L = list(range(9))

def func(n, total):
    interval = 0.001 / (n + 2)
    i = 0
    result = 0
    text = "#{}, est. {:<04.2}s".format(n, interval * total)
    for i in trange(total, desc=text, position=n):
        result += i
        sleep(interval)
    return result

Parallel(n_jobs=4)(delayed(func)(l,10000) for l in L)
```

Reply:
Thx ! 
That's good enough I think. No need to add a dependency on  tqdm.